in-toto-spec.md

```diff
@@ -1011,9 +1035,11 @@ in section 4.3.1.
 The COMMAND field contains the command and its arguments as executed by the
 functionary.
 
-The `"materials"` and `"products"` fields are dictionaries keyed by a file’s
-PATH. Each HASH value is a hash object calculated for the contents of the file
-as described in section 4.2.3.
+The `"materials"` and `"products"` fields are dictionaries keyed by
+ARTIFACT_NAME. For files, this is typically the path of the file. For abstract
```

Comment:
The artifact rules section is still very "path"-centered in its wording. I wonder if it's less confusing to just continue calling the placeholder "PATH" but also refer to ITE-4 here?

Reply:
Took a pass at using names and indicating that while we use glob for patterns, artifact names needn't be paths.